cmd/vulcan-wpscan/main.go

```diff
@@ -55,6 +55,10 @@ func main() {
 		}
 
 		wpScanReport, err := RunWpScan(ctx, logger, target, url, token)
+		if err != nil && err.Error() == NotAWordPressMessage {
+			logger.Warnf("%s", err)
+			return checkstate.ErrAssetUnreachable
+		}
```

Comment:
I think we are misusing the `INCONCLUSIVE` status here. The intent of that status is to detect stale assets so we might notify the end user that some action is required by them (e.g. update or remove). It would be appropriate if we had an asset type like `WordPress` but we don't. Every check ends up `FINISHED` with empty `[]report.Vulnerability` when no issues are found.

Reply:
Right, I see. Suggestion applied here [20267b9](https://github.com/adevinta/vulcan-checks/pull/182/commits/20267b9bd2809f4e13d38857ecd732a4ac46791c)